tests/unittests/distros/test_ifconfig.py

```diff
@@ -38,6 +38,13 @@ def test_vtnet_options(self):
         ifs = Ifconfig().parse(self.ifs_txt)
         assert "txcsum" in ifs["vtnet0"].options
 
+    def test_duplicate_mac(self):
+        """assert that we can have duplicate macs"""
+        ifc = Ifconfig()
+        ifc.parse(self.ifs_txt)
+        ifs_by_mac = ifc.ifs_by_mac()
+        assert len(ifs_by_mac["00:0d:3a:54:ad:1e"]) == 2
```

Comment:
we can do another assert here, that the two devices are different ones, and this isn't an accident or bug in the parser